source/includes/wp-api-v2/_coupons.md

```diff
@@ -4,31 +4,40 @@ The coupons API allows you to create, view, update, and delete individual, or a
-| `minimum_amount`              | string    | Minimum order amount that needs to be in the cart before coupon applies.                                                                              |
-| `maximum_amount`              | string    | Maximum order amount allowed when using the coupon.                                                                                                   |
-| `email_restrictions`          | array     | List of email addresses that can use this coupon.                                                                                                     |
-| `used_by`                     | array     | List of user IDs who have used the coupon. <i class="label label-info">read-only</i>                                                                  |
+|           Attribute           |    Type   |                                                            Description                                                             |
+|-------------------------------|-----------|------------------------------------------------------------------------------------------------------------------------------------|
+| `id`                          | integer   | Unique identifier for the object. <i class="label label-info">read-only</i>                                                        |
+| `code`                        | string    | Coupon code. <i class="label label-info">mandatory</i>                                                                             |
+| `amount`                      | string    | The amount of discount.                                                                                                            |
+| `date_created`                | date-time | The date the coupon was created, in the site's timezone. <i class="label label-info">read-only</i>                                 |
+| `date_modified`               | date-time | The date the coupon was last modified, in the site's timezone. <i class="label label-info">read-only</i>                           |
+| `discount_type`               | string    | Determines the type of discount that will be applied. Options: `fixed_cart`, `percent` and `fixed_product`. Default: `fixed_cart`. |
+| `description`                 | string    | Coupon description.                                                                                                                |
+| `date_expires`                | string    | UTC DateTime when the coupon expires.                                                                                              |
+| `usage_count`                 | integer   | Number of times the coupon has been used already. <i class="label label-info">read-only</i>                                        |
+| `individual_use`              | boolean   | Whether coupon can only be used individually.                                                                                      |
```

Comment:
True if the coupon must be applied individually (other coupons will be removed when applied).